test-clients/src/main/resource/contract/bytecodes/AssociateDissociateContract.bin

```diff
@@ -0,0 +1 @@
+608060405234801561001057600080fd5b506040516105823803806105828339818101604052810190610032919061008d565b806000806101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff160217905550506100ff565b600081519050610087816100e8565b92915050565b60006020828403121561009f57600080fd5b60006100ad84828501610078565b91505092915050565b60006100c1826100c8565b9050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b6100f1816100b6565b81146100fc57600080fd5b50565b6104748061010e6000396000f3fe608060405234801561001057600080fd5b50600436106100365760003560e01c80636945cb3a1461003b578063ec904f1914610045575b600080fd5b61004361004f565b005b61004d61007c565b005b6100793360008054906101000a900473ffffffffffffffffffffffffffffffffffffffff166100a9565b50565b6100a63360008054906101000a900473ffffffffffffffffffffffffffffffffffffffff166101c5565b50565b600080606061016773ffffffffffffffffffffffffffffffffffffffff1663099794e860e01b86866040516024016100e2929190610376565b604051602081830303815290604052907bffffffffffffffffffffffffffffffffffffffffffffffffffffffff19166020820180517bffffffffffffffffffffffffffffffffffffffffffffffffffffffff838183161783525050505060405161014c919061035f565b600060405180830381855af49150503d8060008114610187576040519150601f19603f3d011682016040523d82523d6000602084013e61018c565b606091505b509150915060008261019f5760146101b4565b818060200190518101906101b391906102f6565b5b90508060070b935050505092915050565b600080606061016773ffffffffffffffffffffffffffffffffffffffff166349146bde60e01b86866040516024016101fe929190610376565b604051602081830303815290604052907bffffffffffffffffffffffffffffffffffffffffffffffffffffffff19166020820180517bffffffffffffffffffffffffffffffffffffffffffffffffffffffff8381831617835250505050604051610268919061035f565b600060405180830381855af49150503d80600081146102a3576040519150601f19603f3d011682016040523d82523d6000602084013e6102a8565b606091505b50915091506000826102bb5760146102d0565b818060200190518101906102cf91906102f6565b5b90508060070b935050505092915050565b6000815190506102f081610427565b92915050565b60006020828403121561030857600080fd5b6000610316848285016102e1565b91505092915050565b610328816103b5565b82525050565b60006103398261039f565b61034381856103aa565b93506103538185602086016103f4565b80840191505092915050565b600061036b828461032e565b915081905092915050565b600060408201905061038b600083018561031f565b610398602083018461031f565b9392505050565b600081519050919050565b600081905092915050565b60006103c0826103d4565b9050919050565b60008160070b9050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b60005b838110156104125780820151818401526020810190506103f7565b83811115610421576000848401525b50505050565b610430816103c7565b811461043b57600080fd5b5056fea2646970667358221220d30993dd1118af336ad45e5bfd3de11018abe9ae12bf067ff74ec217b2dfcf3f64736f6c634300060c0033
```

Comment:
We should also include a file named AssociateDissociateContract.sol with the solidity code that was compiled to this bytecode